scripts/mkcloud

```diff
@@ -1218,86 +1124,176 @@ function cloudupgrade_reboot_and_redeploy_clients()
+        use Media from Devel:Cloud:Staging and add test update repositories (except for GM* targets)
+    cephvolumenumber  (default=0)
+        the number of 11GB ceph volumes that will be created per node
+        note: proposal step does NOT contain a ceph proposal, do it manually
+    nodenumber=2    (default 2)
+        set the number of nodes to be created for the cloud (excluding admin node)
+        In HA mode (hacloud=1) the nodes needed for clusters are subtracted from nodenumber; the
+        remaining nodes are compute nodes.
+    nodenumberlonelynode=2    (default 0)
+        set the number of non-crowbar registered nodes to be created in the admin network
+    nodenumbercompute=1  (default $nodenumber resp. 1 in case of hacloud=1)'
+        set the number of compute nodes in a HA setup, the remaining nodes
+        of $nodenumber - $nodenumbercompute are used as HA cluster nodes
+    vcpus=1         (default 1)
+        set the number of CPU cores per compute node
+    adminvcpus=1    (default $vcpus)
+        set the number of CPU cores for admin node
+    admin_node_memory (default 2097152)
+        Set the memory in KB assigned to admin node
+    controller_node_memory (default 4194304)
+        Set the memory in KB assigned to compute nodes
+    compute_node_memory (default 2097152)
+        Set the memory in KB assigned to compute nodes
+    networkingplugin
+        Set the networking plugin to be used by neutron (e.g. openvswitch),
+        if it isn't defined the barclamp-neutron's default is used.
+    networkingmode
+        Set the networking mode to be used by neutron (e.g. gre)
+        if it isn't defined the barclamp-neutron's default is used.
+    keep_existing_hostname=1    (default='')
+        If this option is enabled crowbar_register keeps the existing
+        hostname. When the crowbar_register option is enabled too then
+        it generates and sets random hostnames.
+    debug_mkcloud=1  (default 0)
+        enable debug mode for mkcloud via 'set -x'
+    debug_qa_crowbarsetup=1 (default 0)
+        enable debug mode for qa_crowbarsetup.sh via 'set -x'
+    debug_openstack=1 (default 0)
+        enable debug mode for the openstack components
+        sets debug true in the openstack proposals
+    user_keyfile='path/to/file'
+        path to optional user public ssh keyfile to inject into crowbar to
+        /root/.ssh/authorized_keys file and to provisioner barclamp
+    want_neutronsles12=1 (default 0)
+        if there is a SLE12 node, deploy neutron-l3 role into the SLE12 node
+    install_chef_suse_override='path/to/script'
+        Optional path to an alternate version of install-chef-suse.sh on the mkcloud hostto use
+        instead of the one from the packages.  This will be scp'd to the admin node before use.
+    public_vlan=<id> (default 300)
+        VLAN id for public network
+
+EOUSAGE
     exit 1
+
+# UNDOCUMENTED OPTIONS:
+#
+# virtualcloud
+# cloudfqdn
+# forwardmode
```

Comment:
Actually I did :)
But this commit should do the cleanup now. Lets do the rest in another commit and PR, otherwise we will never be able to merge this PR.